src/runtime/engines/singularity/process.go

```diff
@@ -99,7 +154,7 @@ func (engine *EngineOperations) StartProcess(masterConn net.Conn) error {
 	}
 
 	// Spawn and wait container process, signal handler
-	cmd := exec.Command(args[0], args...)
+	cmd := exec.Command(args[0], args[1:]...)
```

Comment:
What's the purpose of this change here?

Reply:
To avoid redundant first argument, `Command` is not `syscall.Exec` :)